source/macros.tex

```diff
@@ -432,7 +432,7 @@
 \newcommand{\defexposconceptnc}[1]{\ecname{#1}\indexconcept{\idxexposconcept{#1}|idxbfpage}\itcorr[-1]}  % macro length: 18
 
 %% Ranges
-\newcommand{\Range}[4]{\tcode{#1#3,\penalty2000{} #4#2}}
+\newcommand{\Range}[4]{\ensuremath{#1}\tcode{#3}{,}\,\penalty2000{}\tcode{#4}\ensuremath{#2}}
```

Comment:
Should the comma also be in `\ensuremath`?

Reply:
Since math mode cannot see the adjacent stuff, I'm not sure it does anything.

I've done that now:

![range](https://user-images.githubusercontent.com/23412755/155714351-1f2e575f-2eb4-47cc-bc79-64862eedd88c.png)

Reply:
It will at least pick the right maths font, even if that ends up being the same :-)

But yes, the interesting spacing aspects of maths mode probably don't come into play here.

Thanks!